ui/controls.js

```diff
@@ -522,6 +522,29 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     return this.seekBar_ ? this.seekBar_.getValue() : this.video_.currentTime;
   }
 
+  /**
+   * Checking the efficiency of a video
```

Comment:
How full the buffer is would not be described as "efficiency".  Please fix this comment and the title of the PR.  Thanks!